plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -710,7 +711,7 @@ public void testUnsupportedDate()
                     "\\QFailed to insert data: ORA-01841: (full) year must be between -4713 and +9999, and not be 0\n");
             assertQueryFails(
                     format("INSERT INTO %s VALUES (DATE '10000-01-01')", table.getName()),
-                    "\\QFailed to insert data: ORA-01862: the numeric value does not match the length of the format item\n");
+                    "\\QFailed to insert data: ORA-01861: literal does not match format string\n");
```

Comment:
Are we using a wrong format string?

Reply:
I think it's right format because Oracle doesn't support 5 digit year either.

Reply:
Worth adding a code comment.